css/menu.css

```diff
@@ -360,7 +360,8 @@ body.has-menu-left.has-push-menu .fl-viewport-header {
   display: block;
 }
 
-.fl-list-default ul > li.with-icon {
+.fl-list-default ul > li.with-icon,
+.fl-list-default ul > li.with-icon ~ li {
```

Comment:
The `~` selector only targets **subsequent** siblings, and not the preceeding siblings.

See https://jsfiddle.net/z21ponsm/1/ for an example.

This means if only the 3rd list (out of 5) has an icon configured, you'll end up with the 3rd, 4th and 5th padded on the left, but not the 1st and 2nd.